src/Aspire.Hosting.MongoDB/MongoDBDatabaseResource.cs

```diff
@@ -27,7 +27,17 @@ public MongoDBDatabaseResource(string name, string databaseName, MongoDBServerRe
     /// Gets the connection string expression for the MongoDB database.
     /// </summary>
     public ReferenceExpression ConnectionStringExpression
-        => ReferenceExpression.Create($"{Parent}/{DatabaseName}");
+    {
+        get
+        {
+            var builder = new ReferenceExpression.ExpressionInterpolatedStringHandler(10, 2);
```

Comment:
Why `10`?

Reply:
out of date now because I learned about the `ReferenceExpressionBuilder` but it was the length of the scheme and a bit more for a server.